video_player1:
  ros__parameters:
    config_found: true
    publish_frequency_double: 30.0
    loop_play: false
    topic: /video_player1/image
    publish_as_color: true
    filename: /home/maimon/eternarig_ws/src/video_interface/videos/fictrac_bee.mp4
    # filename: /home/maimon/eternarig_ws/src/video_interface/videos/maimon_fruitfly.avi
    # filename: /home/maimon/eternarig_ws/src/video_interface/videos/topview_4arena.avi
    # filename: /home/maimon/src/record_h264/video_20200901_033706_cam18517933-000.mp4
    # filename: /home/maimon/eternarig_ws/src/video_interface/data/test_to_compress.avi
    # loop_play: true

video_saver1:
  ros__parameters:
    config_found: true
    output_fps_double: 30.0
    topic: /video_player1/image
    output_filename: /home/maimon/eternarig_ws/src/video_interface/data/fictrac_bee
    codec: h264 # choose from [xvid, mjpg, h264, raw]
    # output_filename: /home/maimon/eternarig_ws/src/video_interface/data/fruitfly_compressed
    # output_filename: /home/maimon/eternarig_ws/src/video_interface/data/topview_4arena
    # output_filename: /home/maimon/eternarig_ws/src/video_interface/data/test_to_compress_out
